server/src/main/java/org/elasticsearch/persistent/PersistentTasksService.java

```diff
@@ -47,15 +45,14 @@
  */
 public class PersistentTasksService extends AbstractComponent {
 
-    private static final String ACTION_ORIGIN_TRANSIENT_NAME = "action.origin";
```

Comment:
Now that the server has `ThreadContext.stashWithOrigin` and `OriginSettingClient` we can use it. I believe this change is a noop, and the tests seem to agree, but I'd love a second set of eyes on it.